src/pint/derived_quantities.py

```diff
@@ -0,0 +1,864 @@
+"""Functions to compute various derived quantities"""
```

Comment:
It might be good to put a little further explanation here what we mean by "derived quantities"; this docstring goes at the top of the module doc page.

Reply:
Look at the latest (when the docs build). I put in a warning for `utils.py` and added better docstrings for this module.  I did it manually (maybe there is a way to automate that) but it's a useful way of keeping track of the different tools (as we were discussing)

Reply:
We can generate a table, like this, just fine. I'll look into it, and maybe we should file a bug report suggesting that so I don't forget. What I was suggesting was just a short couple of sentences to the effect that this module is for computing various physical pulsar-related quantities from other physical pulsar-related quantities.

Reply:
I think now we have #1105 maybe leave the short summaries out? https://nanograv-pint--1105.org.readthedocs.build/en/1105/_autosummary/pint.utils.html#module-pint.utils

Reply:
Yes, that looks good. Will do.